### PR TITLE
feat(specs): add v2 endpoints for ingestion

### PR DESCRIPTION
**Pull request by shortcuts**
## 🧭 What and Why

🎟 JIRA Ticket: https://algolia.atlassian.net/browse/DI-2663

### Changes included:

Add the push endpoint to the specs

### CODE REVIEW

**Review by millotp — COMMENTED** [review]
nice

File: specs/search/paths/objects/batch.yml [hunk 1]
```diff
@@ -12,53 +12,13 @@ post:
     - Actions are equivalent to the individual API requests of the same name.
   parameters:
     - $ref: '../../../common/parameters.yml#/IndexName'
+  x-codegen-request-body-name: batchWriteParams
```
[the remark this answers is not in the document]
Comment (shortcuts) [reply specs/search/paths/objects/batch.yml:15]: no I thought so for the swift issue but it's a leftover indeed

File: specs/ingestion/paths/tasks/v2/pushTask.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  description: Push a `batch` request payload through the Pipeline. You can check the status of task pushes with the observability endpoints.
+  operationId: pushTask
+  x-acl:
+    - addObject
+    - deleteIndex
+    - editSettings
+  parameters:
+    - $ref: '../../../common/parameters.yml#/pathTaskID'
+  requestBody:
+    description: Request body for creating a task.
```
Comment (millotp) [inline specs/ingestion/paths/tasks/v2/pushTask.yml:14]:
```suggestion
    description: Request body for pushing object into an Algolia index
```
?

**Review by millotp — APPROVED** [review]
gg !

**Pull request merged**